services/src/messaging/src/main/java/org/openkilda/messaging/model/SwitchId.java

```diff
@@ -31,29 +32,31 @@
     private static final long serialVersionUID = 1L;
 
     /**
-     * Switch switchId.
+     * Switch id numerical representation.
      */
-    private final long switchId;
+    private final long switchIdNum;
 
     /**
      * Construct an instance based on the colon separated representation of a switch id.
      */
     public SwitchId(String switchId) {
+        Preconditions.checkArgument(Objects.nonNull(switchId), "Switch id must not be null");
```

Comment:
Objects.nonNull over-complicates the code. "switchId != null" is completely cover the case. Objects.nonNull has been designed to be used as a functional predicate.

Objects.requireNonNull is what you needed: "This method is designed primarily for doing parameter validation in methods and constructors"